app/services/merge/merge_organisations_service.rb

```diff
@@ -88,21 +88,34 @@ def merge_lettings_logs(merging_organisation)
         lettings_log.location = location_to_set if location_to_set.present?
       end
       lettings_log.owning_organisation = @absorbing_organisation
-      lettings_log.skip_dpo_validation = true
-      lettings_log.save!
+      lettings_log.managing_organisation = @absorbing_organisation if lettings_log.managing_organisation == merging_organisation
```

Comment:
do we want to add this to sales logs too? or is the sales log managing org behaviour not merged yet?

Reply:
Yeah, we do, I'll add it as part of that PR cause it's still in PO

Reply:
Oh wait, hold on, we're already doing that. All good 😂